pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -599,11 +601,26 @@ func (p *PriorityQueue) AssignedPodAdded(pod *v1.Pod) {
 	p.lock.Unlock()
 }
 
```

Comment:
Please add comments here from my patch on slack to clearly define the intent of this function.
```go
// isPodResourcesResizedDown returns true if a pod CPU and/or memory resize request has been
// admitted by kubelet, is 'InProgress', and results in a net sizing down of updated resources.
// It returns false if either CPU or memory resource is net resized up, or if no resize is in progress.
````